standards/02-Software/05-Contribute.md

```diff
@@ -131,29 +133,28 @@ By participating in this project, you are expected to abide by our [Code of Cond
 
-- **Getting Help:** If you need assistance with Overture, please create a [new discussion in our support category](https://github.com/overture-stack/docs/discussions/categories/support).
-  - Before creating a new discussion, please search existing discussions to see if your question has already been answered.
-- **Feature Requests & Proposals:** We love hearing your ideas for improving Overture! Before making a feature request, please check our [**feature roadmap**](https://github.com/orgs/overture-stack/projects/11/views/1) to see if your idea is already planned.
-  - If your idea isn't on the roadmap, feel free to [**submit a feature request**](https://github.com/overture-stack/docs/discussions/categories/ideas) by creating a new discussion in our Ideas category 
-- **Report a Potential Bug:** We use GitHub Issues primarily for tracking confirmed bugs and ticketing development tasks. If you come across a potential bug or issue, please first post it to our [**GitHub support discussion forum**](https://github.com/overture-stack/docs/discussions/categories/support).
-  - This allows us to confirm the issue and gather more information if needed. If we determine that further development is required, we will create and tag you into a GitHub Issue from your discussion post.
+-   **Getting Help:** If you need assistance with Overture, please create a [new discussion in our support category](https://github.com/overture-stack/docs/discussions/categories/support).
+    -   Before creating a new discussion, please search existing discussions to see if your question has already been answered.
+-   **Feature Requests & Proposals:** We love hearing your ideas for improving Overture! Before making a feature request, please check our [**feature roadmap**](https://github.com/orgs/overture-stack/projects/11/views/1) to see if your idea is already planned.
+    -   If your idea isn't on the roadmap, feel free to [**submit a feature request**](https://github.com/overture-stack/docs/discussions/categories/ideas) by creating a new discussion in our Ideas category
+-   **Report a Potential Bug:** We use GitHub Issues primarily for tracking confirmed bugs and ticketing development tasks. If you come across a potential bug or issue, please first post it to our [**GitHub support discussion forum**](https://github.com/overture-stack/docs/discussions/categories/support).
+    -   This allows us to confirm the issue and gather more information if needed. If we determine that further development is required, we will create and tag you into a GitHub Issue from your discussion post.
 
 ## Our Development Process
 
-We use GitHub issues and pull requests for communication related to code changes. 
+We use GitHub issues and pull requests for communication related to code changes.
 
 ### Branch Organization
 
 We use the following standard branches:
```

Comment:
Your update in the copy paste dropdown section should also be added here, for ref this is the page you are currently updating: https://docs.overture.bio/docs/standards/Software/Contribute

You will also want to update the contributing.md at the root of this repo which gets rendered here: https://docs.overture.bio/docs/contribution